collect_app/src/test/java/org/odk/collect/android/activities/MainMenuActivityTest.java

```diff
@@ -26,12 +26,13 @@
 import dagger.Provides;
 
 import static android.os.Environment.MEDIA_MOUNTED;
+import static androidx.lifecycle.Lifecycle.State.DESTROYED;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.equalTo;
 import static org.robolectric.Shadows.shadowOf;
 import static org.robolectric.annotation.LooperMode.Mode.PAUSED;
 
-@RunWith(RobolectricTestRunner.class)
+@RunWith(AndroidJUnit4.class)
```

Comment:
So is this a no-op or am I missing something?

Reply:
I think so. The docs seem to make a distinction http://robolectric.org/androidx_test/. I guess again for me I find it useful to know that this is an "Android X" test. Do you think it's confusing?

Reply:
It's not a big deal but I'd like to have a sense of when I should explicitly specify a runner or not.

Reply:
I guess I think for these ones there should always be a runner to show that we're not just using JUnit. It's a little wierd because anything in `androidTest` automatically uses this runner but that's a really strange concept for anyone used to JUnit (not even Spring does that).

Reply:
To summarize: for tests in `test` that don't use Robolectric, then it's just JUnit and not specifying an explicit runner makes sense. But if they're in `androidTest` and use the androidx `AndroidJUnit4` runner, we should make that explicit even though it's the default because it's a weird default?

Reply:
My gut tells me that's the right thing yeah.

Reply:
I appreciate your gut for sharing a clear rule to follow. 👍